.env.test

```diff
@@ -0,0 +1,15 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=fake-secret-key
+
+TRELLO_API_KEY=TESTINGAPIKEY
+TRELLO_API_TOKEN=TESTINGAPIKEYTOKEN
+TRELLO_BOARD_ID=658400a65cbc3ab26e171cb0
+TRELLO_BOARD_LIST_ID_TODO=658400a6795546f35c80c673
+TRELLO_BOARD_LIST_ID_DONE=658400a77be14947511810e0
```

Comment:
These aren't sensitive, but we might as well swap them for obviously fake values anyway; one reason being that the more fake values we use by default, the less likely we accidentally end up in a situation where our tests start affecting non-test data (and when you consider how common "intern accidentally emails production" stores are - this is a risk worth considering!)